lib/UserManagement.php

```diff
@@ -128,6 +128,60 @@ public function authenticateUserWithMagicAuth($clientId, $code, $userId, $ipAddr
         return Resource\UserResponse::constructFromResponse($response);
     }
 
+    /**
+     * Authenticate with TOTP
+     *
+     * @param string $clientId This value can be obtained from the Configuration page in the WorkOS dashboard.
+     * @param string $pendingAuthenticationToken
+     * @param string $authenticationChallengeId
+     * @param string $code
+     * @throws Exception\WorkOSException
+     *
+     * @return \WorkOS\Resource\UserResponse
+     */
+
+    public function authenticateUserWithTotp($clientId, $pendingAuthenticationToken, $authenticationChallengeId, $code)
```

Comment:
```suggestion
    public function authenticateWithTotp($clientId, $pendingAuthenticationToken, $authenticationChallengeId, $code)
```
Could we change these methods to match the fact we are in a `Users` stack already, so we'll follow the principle. I'll add the tickets on the other SDKs, but we could do it here already. 🙇